File: classes/how_script/hello.rb
# hello.rb
puts "hello"

# but even cooler
myname = "hunter"
puts "hello #{myname}"

# even fancier

def name_printer (myname)
    
  if myname.class == String
     puts "hello my name is #{myname}"
  elsif myname.class == Integer
     puts "beep boop I am robot, my serial number is #{myname}"
  else
     puts "name input '#{myname}' is date type'#{myname.class}'"
     raise "Input for name can only be a string or integer"

  end
end

name_printer "hunter"
name_printer 12341234
name_printer 1.1

File: classes/how_script/hello.py
# hello.py
print('hello')

# but even cooler
myname = 'hunter'
print(f'hello from {myname}')

# even fancier
def name_printer(myname):

    if type(myname) == str:
        print(f'hello my name is {myname}')
    elif type(myname) == int:
        print(f'beep boop I am robot, my serial number is {myname}') 
    else:
        print(f"name input '{myname}' is date type'{type(myname)}'\n")
        raise AssertionError(f"Input for name can only be a string or integer")

name_printer('hunter')
name_printer(12341234)
name_printer(1.1)
